python/neuroglancer/pipeline/scripts/generate_manifests.py

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/python
 
 """
+WARNING: This script is outdated. Try using MeshManifestTasks 
```

Comment:
Why not removing this entirely if outdated?

Reply:
I think it might have been useful to me for a small dataset. I'm a little hesitant to remove scripts that can help researchers working with small volumes. For instance, Tommy has asked me on one or two occasions for how to do things without having to run the pipeline. What do you think?